chia/wallet/notification_store.py

```diff
@@ -44,7 +44,7 @@ async def create(
                 "CREATE TABLE IF NOT EXISTS notifications(" "coin_id blob PRIMARY KEY," "msg blob," "amount blob" ")"
             )
 
-            await conn.execute("CREATE INDEX IF NOT EXISTS coin_id_index on notifications(coin_id)")
+            await conn.execute("DROP INDEX IF EXISTS coin_id_index")
```

Comment:
this is redundant because `coin_id` is already the `PRIMARY KEY` of the table, and has an implicit index.
I think it would make sense to leave a comment here explaining the history a bit, and maybe in which version we switched from creating this index to dropping it. because at some point in the future we may want to remove this line entirely

Reply:
Comment added